tests/phpunit/CRM/Contribute/Form/Task/StatusTest.php

```diff
@@ -62,6 +116,10 @@ public function testUpdatePendingContribution() {
     $this->assertEquals(date("Y-m-d"), date("Y-m-d", strtotime($updatedContribution['receive_date'])));
     $this->assertNotEquals("00:00:00", date("H:i:s", strtotime($updatedContribution['receive_date'])));
     $this->assertEquals('Completed', $updatedContribution['contribution_status']);
+
+    $msg = $mut->getMostRecentEmail();
+    $this->assertNotEmpty($msg);
+    $mut->stop();
```

Comment:
Similarly what you're testing here is that it DIDN'T SEND, so lines 120-121 should be `$mut->assertMailLogEmpty();`

Reply:
You are right